oidc-controller/api/core/oidc/issue_token_service.py

```diff
@@ -32,7 +32,7 @@ class Token(BaseModel):
 
     @classmethod
     def get_claims(
-        cls, pres_exch: Dict, auth_session: AuthSession, ver_config: VerificationConfig
```

Comment:
This parameter is no longer used. The auth session model generates it as a property.